fabric-data-attachment-api-v1/src/main/resources/fabric-data-attachment-api-v1.mixins.json

```diff
@@ -0,0 +1,15 @@
+{
+  "required": true,
+  "package": "net.fabricmc.fabric.mixin",
```

Comment:
```suggestion
  "package": "net.fabricmc.fabric.mixin.attachment",
```

Reply:
Isn't it convention to have the mixins in a different folder for each module? Must've looked wrong at other modules

Reply:
Keep the mixins in the same package, you can just clean up the json a little.

Reply:
oh right i'm stupridr